osbuild/main_cli.py

```diff
@@ -27,8 +29,16 @@ def parse_manifest(path):
     if path == "-":
         manifest = json.load(sys.stdin)
     else:
-        with open(path) as f:
-            manifest = json.load(f)
+        if not os.path.exists(path) and path.startswith("http"):
```

Comment:
This overloads the manifest-argument without really defining it properly, does it? Like this feature now depends on whether a local-file with a given URL exists or not. That sounds like a very odd behavior. I would not like a command to change behavior based on the existence of a local file (in an unintended way).

My suggestion would be to drop `os.path.exists()`, and require callers to use absolute paths or `./` as leading designator to avoid the HTTP handling.

Additionally, can you check for `http:` and `https:` instead of `http`?